app/containers/mainPanel.js

```diff
@@ -235,15 +235,22 @@ class MainPanel extends Component {
 
     const isFavoriteChannel = settings['favorite-channels'] && settings['favorite-channels'].includes(cabal.channel)
 
+    function getChannelName () {
+      const userKey = Object.keys(cabal.users).find((key) => key === cabal.channel)
+      const pmChannelName = cabal.users[userKey]?.name ?? cabal.channel
```

Comment:
```suggestion
      const pmChannelName = cabal.users[userKey]?.name ?? cabal.channel.slice(0,8)
```

so as to not get the entire gazillion character long public key